installing-hck-ci-from-scratch.txt

```diff
@@ -51,59 +53,48 @@ Configure Jenkins
 
 5. Build the docker image: docker build . -t hck-ci/jenkins -f jenkins.dockerfile
 
-6. Run docker container (including restart option): docker run -d --restart unless-stopped -p 8080:8080 -p 50000:50000 -v jenkins_home:/var/jenkins_home hck-ci/jenkins
+6. Run docker container (detached with auto restart): 
+   docker run -d --restart unless-stopped -p 8080:8080 -p 50000:50000 -v jenkins_home:/var/jenkins_home hck-ci/jenkins
+   
+7. Find the container id: docker ps
 
-7. Find the ID of your container: docker ps
+8. Open a shell inside the container with: exec -i -t <container_id> /bin/bash
 
-8. Create a SSH private key in the container: docker exec -it <containerid> ssh-keygen -t rsa
+9. Create a new private key inside the container: ssh-keygen -t rsa
 
-9. Copy the public key from the container to the host authorized_keys:
-
-   docker exec -it <containerid> ssh-copy-id -i /var/jenkins_home/.ssh/id_rsa.pub <username>@<host>
-
-   If this doesn't work:
-   cat /var/lib/docker/volumes/jenkins_home/_data/.ssh/id_rsa.pub
-   vi ~/.ssh/authorized_keys
-
-   Append the key to the end of file.
-
-   You should connect from docker container to host as least once to accept the key:
-   Run docker container (including restart option): docker run -it --entrypoint /bin/bash hck-ci/jenkins
-
-   To test that ssh connection works, run Jenkins container in interactive mode and try to ssh to host (to stop running container use: docker stop <container ID>):
-   docker run -it -p 8080:8080 -p 50000:50000 -v jenkins_home:/var/jenkins_home --entrypoint /bin/bash hck-ci/jenkins
-   ssh -l <host_username> <host> -i /var/jenkins_home/.ssh/id_rsa
+10. Copy the public key to the host: ssh-copy-id -i ~/.ssh/id_rsa user@host
+    * If you can't ssh the host detach from container and run:
+    cat /var/lib/docker/volumes/jenkins_home/_data/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+    
+11. Make sure ssh to host works and detach from container (ctrl-d)
```

Comment:
make sure you can ssh to the host.

Also don't remove the example of command line